packages/worker/src/worker-options.ts

```diff
@@ -210,19 +280,37 @@ export interface WorkerOptions {
   /**
    * The number of Workflow isolates to keep in cached in memory
    *
-   * Cached Workflows continue execution from their last stopping point.
-   * If the Worker is asked to run an uncached Workflow, it will need to replay the entire Workflow history.
-   * Use as a dial for trading memory for CPU time.
+   * Cached Workflows continue execution from their last stopping point. If the Worker is asked to run an uncached
+   * Workflow, it will need to fetch and replay the entire Workflow history.
+   *
+   * **When `reuseV8Context` is disabled** — Most users are able to fit at least 250 Workflows per GB of available
+   * memory. The major factors contributing to a Workflow's memory weight are the size of allocations made by the
+   * Workflow itself and by all loaded librairies (including the Node JS builtin context), as well as the size of the
+   * Workflow bundle (code and source map). For the SDK test Workflows, we managed to fit 750 Workflows per GB. Your
```

Comment:
And the size of inputs (stored in memory state). Note that there's a bug in Core where it does not discard payloads from already replayed events.